scripts/tag_autocomplete_helper.py

```diff
@@ -150,4 +182,15 @@ def on_ui_settings():
     shared.opts.add_option("tac_extra.extraFile", shared.OptionInfo("None", "Extra filename", gr.Dropdown, lambda: {"choices": csv_files_withnone}, refresh=update_tag_files, section=TAC_SECTION))
     shared.opts.add_option("tac_extra.onlyAliasExtraFile", shared.OptionInfo(False, "Extra file in alias only format", section=TAC_SECTION))
 
-script_callbacks.on_ui_settings(on_ui_settings)
\ No newline at end of file
+script_callbacks.on_ui_settings(on_ui_settings)
+
+
+class Script(scripts.Script):
+    def title(self):
+        return "autogen"
+
+    def show(self, is_img2img):
+        return scripts.AlwaysVisible
+
+    def ui(self, is_img2img):
+        get_ext_wildcard_tags()
```

Comment:
The reason for the double run is that you call `get_ext_wildcard_tags()` both on line 142 and here in the ui def in 196.
The script is already run on load by default and doesn't interact with the UI in any way, so there's no need for the whole Script class setup.
```suggestion
```

Reply:
For script reload without restarting completely you don't need the script definition either, you can just use the restart button on the bottom of the settings page which works fine for most things.

Either way, not really a concern here outside of faster debugging since the yaml wildcards aren't changing on the fly.